src/v2/Components/ArtworkFilter/ArtworkFilters/ArtistNationalityFilter.tsx

```diff
@@ -11,12 +11,12 @@ export const ArtistNationalityFilter: React.FC<ArtistNationalityFilterProps> = (
 }) => {
   return (
     <ResultsFilter
+      expanded={expanded}
```

Comment:
We're at the point where all of these filter components are generic enough to be generated from an array of config objects. Not sure if thats worth it or not, cause this is pretty readable and discoverable